voting/src/voting.rs

```diff
@@ -198,6 +198,27 @@ impl TxOwner for Context {
                 }
                 Ok(())
             }
+            VOTE_TX_TYPE => {
+                let tx: VoteTransaction = serde_cbor::from_slice(&transaction.body()).map_err(|_| todo_fixthis)?;
+                let vote_paper_id = tx.action.vote_paper_id;
+                let choice = tx.action.choice;
+                let voter_signature = tx.action.voter_signature;
+
+                if choice != FAVOR || choice != AGAINST || choice != ABSENTION {
+                    return Err(ExecuteError::InvalidChoice).map_err(|_| todo_fixthis)
+                }
+                let mut vote_paper: VotePaper =
+                    serde_cbor::from_slice(&self.storage().get(vote_paper_id.0.as_ref()).unwrap())
+                        .map_err(|_| ExecuteError::VotePaperNotFound)
+                        .map_err(|_| todo_fixthis)?;
```

Comment:
Please take care when you use `unwrap`.

```suggestion
                    serde_cbor::from_slice(&self.storage().get(vote_paper_id.0.as_ref()).map_err(|_| ExecuteError::VotePaperNotFound)?)
                        .unwrap()
                        .map_err(|_| todo_fixthis)?;
```